### DISCUSSION TITLE
Version 1.28

### DISCUSSION THREAD

**Discussion by johnkeel-thork** [johnkeel-thork/DMD2#22] [Change Log]
- Added Directional Arrows over GPX lines
- Added many Asian maps
- Improved roads contrast on topo maps

**Comment by nezgrath** [comment]
If you'll have the time could you also modify the standard road map view to have higher contrast roads?

**Reply by johnkeel-thork** [reply]
Yes will do.
And probably in the future I will add a sub-option for each map type (road/topo) to enable "DAY HIGH-CONTRAST"(with even higher contrast and thicker lines) and a "NIGHT MODE".

**Reply by nezgrath** [reply]
Cool. As for high contrast - even better, it would have probably been my next question.
